azure-pipelines.yml

```diff
@@ -179,49 +171,3 @@ jobs:
           PYTHON="$(python.pythonLocation)\\python.exe"
           $PYTHON -m asv check -v -E existing
         displayName: Check benchmarks
-
-      - bash: |
-          set -ex
-          export PYTHON="$(python.pythonLocation)\\python.exe"
-
-          # Install the doc dependencies
-          $PYTHON -m pip install ${PIP_FLAGS} -r requirements/docs.txt
-          $PYTHON -m pip list
-
-          # Build the documentation
-          choco install optipng
-          export SPHINXCACHE=${AGENT_BUILDDIRECTORY}/.cache/sphinx
-          export SPHINXOPTS="-W -j auto"
```

Comment:
I think this is currently the only place in our CI where we fail on warnings for Sphinx. We should make sure to add that flag in another job.

To be honest, I would prefer if we discuss and deal with this separately since. See also https://github.com/scikit-image/scikit-image/pull/7658 that proposes a different route.

Reply:
I emailed GH about the faster runners earlier today, and will be happy to reinstate the Windows docs build if we are able to get those going. The 4x build time is a major blocker right now for getting CI build times down. I saw the differently proposed route, and will be happy to add that if you and @jarrodmillman come to an agreement.

Reply:
Noted about the warnings, thanks.